sample_tracking/app/top_bar.tsx

```diff
@@ -35,14 +37,14 @@ export default function TopBar() {
     useEffect(() => {
         if (!userData) {
             onAuthStateChanged(auth, (user) => {
-                if (!user) {
-                    router.push('/login');
-                } else {
+                if (user) {
                     setUserData({
                         name: user.displayName!,
                         photoUrl: user.photoURL!,
                         email: user.email!,
                     } as UserData)
+                } else {
+                    setUserData(null);
```

Comment:
i would have thought /login would still be valid but include the header? no?

Reply:
Yes this is just saying that there is no logged in user so we won't show the user profile image/letter in the top right.

Reply:
We can test with the link created in the PR. But I've tested with both seeing the top bar when logged in and logged off and the profile image is shown only when logged in.